src/components/PaginationRow/PaginationRow.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ClassNames from 'classnames';
+
+import { MenuItem, DropdownButton, Icon } from '../../index';
+
+const ArrowIcon = props => {
+  const name = `angle-${props.name}`;
+  return <Icon type="fa" name={name} className="i" />;
```

Comment:
Can you move the `ArrowIcon` into `src/components/PaginationRow/InnerComponents/ArrowIcon.js` as described here:
https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#code-consistency

Reply:
i think we can expose it ;)